tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/fly.ts

```diff
@@ -0,0 +1,41 @@
+import { createLanguagePerk, Species } from "./base";
+
+const Fly: Species = {
+  description: "With no official documentation or knowledge of the origin of \
+    this species, they remain a mystery to most. Any and all rumours among \
+    Nanotrasen staff regarding flypeople are often quickly silenced by high \
+    ranking staff or officials.",
+  features: {
+    good: [{
+      icon: "grin-tongue",
```

Comment:
```suggestion
      icon: "head-side-cough",
```

![image](https://user-images.githubusercontent.com/35135081/134835336-08d632c7-9b78-4984-811f-c19b49c32db3.png)

i just like this one better

Reply:
wait we dont have it from the looks of it

Reply:
gotcha